dsn/src/builder.rs

```diff
@@ -331,39 +317,22 @@ impl Dsn {
         destructors.add_items_to_drop(on_new_listener)?;
 
         let on_peer_info = node.on_peer_info(Arc::new({
-            let archival_storage_info = farmer_archival_storage_info.clone();
-
             move |new_peer_info| {
                 let peer_id = new_peer_info.peer_id;
                 let peer_info = &new_peer_info.peer_info;
 
-                if let PeerInfo::Farmer { cuckoo_filter } = peer_info {
-                    archival_storage_info.update_cuckoo_filter(peer_id, cuckoo_filter.clone());
-
+                if let PeerInfo::Farmer { placeholder: _ } = peer_info {
                     tracing::debug!(%peer_id, ?peer_info, "Peer info cached",);
```

Comment:
Not really, nothing to cache anymore. We probably don't care about this event just like we do not subscribe to disconnections below.